config/deps/endpoint.exs

```diff
@@ -1,9 +1,12 @@
 import Config
 
+default_key_base = "yK6hUINZWlq04EPu3SJjAHNDYgka8MZqgXZykF+AQ2PvWs4Ua4IELdFl198aMvw0"
+secret_key_base = System.get_env("SITE_SECRET_KEY_BASE", default_key_base)
```

Comment:
this should be handled in the `runtime.exs` config so it's not just set at compile time